bigbone-rx/src/main/kotlin/social/bigbone/rx/RxReportMethods.kt

```diff
@@ -13,10 +14,17 @@ import social.bigbone.api.method.ReportMethods
 class RxReportMethods(client: MastodonClient) {
     private val reportMethods = ReportMethods(client)
 
-    fun fileReport(accountId: String, statusIds: List<String>, comment: String): Single<Report> {
+    fun fileReport(
```

Comment:
Now as parameters can also be nullable, we should do the following to keep the API usable for Java developers:
- Add the `@JvmOverloads` annotation to the method
- Put all the nullable parameters to the end of the list, so that we get the best out of the automatically generated Java method overloads that are automatically generated by using `@JvmOverloads`.

Above changes need to also be added to `ReportMethods.kt`.

Thank you very much in advance.